recovery/recovery_test.go

```diff
@@ -0,0 +1,26 @@
+package recovery
+
+import (
+	"bytes"
+	"github.com/socialpoint-labs/bsk/logx"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestWithRequestResponseLogs(t *testing.T) {
+	t.Parallel()
+	a := assert.New(t)
+
+	t.Run("panic handler serializes panic", func(t *testing.T) {
+		w := bytes.NewBufferString("")
+		logger := logx.New(logx.WriterOpt(w))
+		blankExitFunc := func() {
+			a.Contains(w.String(), "test panicking")
```

Comment:
you could also raise a flag here and make sure this code was actually run by checking it in the end, same as you did in grpcx.